src/reducers/index.js

```diff
@@ -12,9 +12,12 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return { count: state.count + 1 };
```

Comment:
Nice job with your reducers. I would use the `++ --` syntax to make this friendlier toward anyone working on your code.